README.md

```diff
@@ -0,0 +1,53 @@
+# Welcome to cascade 👋
+[![Documentation](https://img.shields.io/badge/documentation-yes-brightgreen.svg)](https://github.com/the-dr-lazy/cascade/wiki)
+[![Twitter: the\_dr\_lazy\_](https://img.shields.io/twitter/follow/the\_dr\_lazy\_.svg?style=social)](https://twitter.com/the\_dr\_lazy\_)
+https://github.com/the-dr-lazy/cascade/blob/master/LICENSE
+> A project management service just to show that real-world Haskell is awesome.
+
+### 🏠 [Homepage](https://github.com/the-dr-lazy/cascade)
+
+### ✨ [Demo](www.demo.com)
+
+## Install
+
+```sh
+install command
+```
+
+## Usage
+
+```sh
+how to use it
+```
+
+## Run tests
+
+```sh
+how to test it
+```
+
+## Author
+
+👤 **Mohammad Hasani**
+
+* Website: https://the-dr-lazy.github.io/
+* Twitter: [@the\_dr\_lazy\_](https://twitter.com/the\_dr\_lazy\_)
+* Github: [@https:\/\/github.com\/the-dr-lazy](https://github.com/https:\/\/github.com\/the-dr-lazy)
+* LinkedIn: [@https:\/\/www.linkedin.com\/in\/the-dr-lazy](https://linkedin.com/in/https:\/\/www.linkedin.com\/in\/the-dr-lazy)
```

Comment:
We should add the other collaborators of the project too.
For template, we can use the [Authors section of the-dr-lazy/purescript-monarch README](https://github.com/the-dr-lazy/purescript-monarch#authors).